### DISCUSSION TITLE
unable to open storage in Android

### DISCUSSION THREAD

**Discussion by JeromTWinL** [ppy/osu#21361] [Q&A] [answered]
Hi, so i just wanna create a beatmap, i tried to browse to /sdcard in osu!lazer but it shows the content of / ,but i tried osu! Lazer in my Xiaomi Tab, i can browse to /sdcard, but it's not necessary to use my tab, any fixes for my phone to access /sdcard?

Android Version: 10 Q

**Comment by nagi-desuuu** [comment, marked as answer]
https://github.com/ppy/osu/issues/11911
